src/main/java/com/minecolonies/coremod/commands/colonycommands/CommandRaidNow.java

```diff
@@ -36,6 +41,32 @@ public int onExecute(final CommandContext<CommandSource> context)
         return 0;
     }
 
+    /**
```

Comment:
This method and the command build seems to be dupe code to me, might be worth to merge "raid now" and "raid tonight" into one command with a boolean flag.
At least try to make common logic abstract.